docker_base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a base container we use. It has pgsql, redis and Python 3.5,
+# plus a user named qabel and locales.
+
+FROM ubuntu:16.04
```

Comment:
can't we get a smaller base image? How big is the resulting image and how much of that is consumed by the base image?